modules/monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus.adoc

```diff
@@ -40,24 +39,27 @@ You can only use bearer token authentication to access the federation endpoint.
 
 .Procedure
 
-. Retrieve the bearer token:
+. Retrieve the bearer token by running the following the command:
 +
 [source,terminal]
 ----
 $ TOKEN=`oc whoami -t`
```

Comment:
```suggestion
$ TOKEN=$(oc whoami -t)
```
for consistency, it's better to update to it, see the same command in https://docs.openshift.com/container-platform/4.15/monitoring/accessing-third-party-monitoring-apis.html